content/source/docs/cloud/workspaces/agent.html.md

```diff
@@ -0,0 +1,175 @@
+
+Additionally, the agent must also be able to communicate with any services required by the Terraform code it is executing. This includes the Terraform releases distribution service, [releases.hashicorp.com](https://releases.hashicorp.com) (supported by [Fastly](https://api.fastly.com/public-ip-list)), as well as any provider APIs. The services which run on these IP ranges are described in the table below.
+
+Hostname | Port/Protocol | Directionality | Purpose
+--|--|--|--
+app.terraform.io | tcp/443, HTTPS | Outbound | Polling for new workloads and providing status updates
+releases.hashicorp.com | tcp/443, HTTPS | Outbound | Updating agent components and downloading Terraform binaries
+archivist.terraform.io | tcp/443, HTTPS | Outbound | Blob Storage
+
+### Operational Considerations
+
+The agent is distributed as a standalone binary which can be run on any supported system. By default, the agent will run in the foreground as a long-running process that will continuously poll for workloads from Terraform Cloud. We strongly recommend pairing the agent with a process supervisor to ensure that it is automatically restarted in case of an error.
+
+Agents do not guarantee a clean working environment per Terraform execution. Each execution is performed in its own temporary directory with a clean environment, but references to absolute  file paths or other machine state may cause interference between Terraform executions. We strongly recommend that you write your Terraform code to be stateless and idempotent. You may also want to consider using [single-execution mode](#optional-configuration-single-execution-mode) to ensure your agent only runs a single workload.
+
+### Updating
+
+The agent will automatically receive updates for the agent only. Administrators are required to update the host operating system and all other software.
+
+### Security Considerations
+
+Agents should be considered a global resource within an organization. Once configured, any workspace owner may configure their workspace to target the organization's agents. This may allow a malicious workspace to access state files, variables, or code from other workspaces targeting the same agent, or access sensitive data on the host running the agent. For this reason, we recommend carefully considering the implications of enabling agents within an organization, and restricting access to your organization to only trusted parties.
+
+### Limitations
+
+Agents are designed to allow you to run Terraform operations from a Terraform Cloud workspace on your private infrastructure. The following use cases are not supported by agents:
+
+- Connecting to private infrastructure from Sentinel policies using the [`http` import](https://docs.hashicorp.com/sentinel/imports/http/)
```

Comment:
`http import` comes out looking really weird in the rendered text, is this formatting necessary?